Explorer/Assets/DCL/Interaction/PlayerOriginated/Components/IgnoreInteractionComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace DCL.Interaction.PlayerOriginated.Components
+{
+    /// <summary>
+    /// Allows to disable avatars mouse interactions
```

Comment:
Maybe if this is a bit more specific it can be less confusing, since one can think this is to disable ANY interaction from any player, maybe change this comment to:
`/// Allows to disable mouse interactions on player avatars` or similar

Reply:
Done!